### PR TITLE
Fix error listing addresses

### PR DESCRIPTION
**Pull request by Tommytrg**

### CODE REVIEW

File: src/components/Addresses.vue [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default {
   },
   data() {
     return {
-      selectedIndex: this.addresses.length - 1 || null,
+      selectedIndex: this.addresses.length - 1 >= 0 ? this.addresses.length - 1 : -1,
```
Comment (aesedepece) [inline src/components/Addresses.vue:52]: The expression `this.addresses.length` does always fulfill the `>= 0` requirement, while the base value is `-1`, so the whole expression actually equals to this:
```suggestion
      selectedIndex: this.addresses.length - 1,
```

File: src/components/Send.vue [hunk 2]
```diff
@@ -148,6 +148,15 @@ export default {
     Alert,
   },
   data() {
+    const enoughFunds = (rule, value, callback) => {
```
Comment (aesedepece) [inline src/components/Send.vue:151]: What's `rule` here?

File: src/components/Send.vue [hunk 3]
```diff
@@ -175,6 +185,10 @@ export default {
   },
   computed: {
     ...mapState({
+      availableBalance: state => {
+        // TODO: change for avaiable when wallet return it
```
Comment (aesedepece) [inline src/components/Send.vue:189]:
```suggestion
        // TODO: change for available when wallet returns it
```

File: tests/unit/src/components/Addresses.spec.js [hunk 4]
```diff
@@ -81,6 +81,24 @@ describe('Addresses.vue', () => {
         }),
       })
 
+      expect(wrapper.find(AddressInformation).exists()).toBe(true)
+    })
+    it("should render properly if there aren't addresses", () => {
```
Comment (aesedepece) [inline tests/unit/src/components/Addresses.spec.js:86]:
```suggestion
    it("should render an empty state if there aren't addresses", () => {
```

File: src/components/Send.vue (hunk 2 above)
Comment (Tommytrg) [reply src/components/Send.vue:151]: This function is used by `Element` to define [custom validation rules](https://element.eleme.io/#/en-US/component/form#custom-validation-rules). `rule` contains information about the validations that you specified in the template such as the type or the the validator function for that type.

File: src/components/Addresses.vue (hunk 1 above)
Comment (Tommytrg) [reply src/components/Addresses.vue:52]: Ashamed :no_mouth:

**Comment by Tommytrg** [comment]
This is already fixed

**Pull request closed without merging**